src/sonic-config-engine/tests/sample_output/py2/buffers-dell6100.json

```diff
@@ -637,4 +637,16 @@
             "profile" : "[BUFFER_PROFILE|egress_lossy_profile]"
         }
     }
+,
```

Comment:
Indent? and move to above line.

Reply:
will add an indent. As [it won't be moved to the if block](https://github.com/Azure/sonic-buildimage/pull/6194#discussion_r541484608), we don't need to move to the above line.